zap/src/main/dist/db/mysql.properties

```diff
@@ -93,7 +93,7 @@ history.ps.insertstd		= INSERT INTO HISTORY (SESSIONID, HISTTYPE, TIMESENTMILLIS
 REQBODY, RESHEADER, RESBODY, TAG, STATUSCODE, NOTE, RESPONSEFROMTARGETHOST) VALUES (?, ? ,?, ?, ?, ?, ?, ? ,? , ?, ?, ?, ?, ?)
 history.ps.insertnost		= INSERT INTO HISTORY (SESSIONID, HISTTYPE, TIMESENTMILLIS, TIMEELAPSEDMILLIS, METHOD, URI, REQHEADER,\
 REQBODY, RESHEADER, RESBODY, TAG, NOTE, RESPONSEFROMTARGETHOST) VALUES (?, ? ,?, ?, ?, ?, ?, ? ,? , ? , ?, ?, ?)
-history.ps.addtag			= ALTER TABLE HISTIRY ADD COLUMN TAG VARCHAR(32768) DEFAULT ''
+history.ps.addtag			= ALTER TABLE HISTORY ADD COLUMN TAG VARCHAR(32768) DEFAULT ''
```

Comment:
This scares me

Reply:
It shouldn't, the table already has the TAG column.